### PR TITLE
[ui] Parallelize ember tests

### PR DESCRIPTION
**Pull request by philrenaud**
Uses Ember Exam to parallelize tests; includes a few fixes for tests that are affected by leaks more than others. Surely more'll pop up eventually and we'll patch them when they do.

### CODE REVIEW

**Review by gulducat — APPROVED** [review]
not knowing JS well, I'm only vaguely aware of what this is doing, but LGTM!

**Review by tgross — APPROVED** [review]
LGTM!

File: ui/tests/acceptance/allocation-detail-test.js [hunk 1]
```diff
@@ -737,6 +739,13 @@ module('Acceptance | allocation detail (services)', function (hooks) {
 
   test('Allocation has a list of services with active checks', async function (assert) {
     await visit('jobs/service-haver@default');
+    console.log('+++++ LOGGING ALLOC DETAILS +++++');
+    console.log('Allocs found: ', findAll('tr[data-test-allocation]').length);
+    console.log('(locally, 1)');
+    await percySnapshot(assert);
+
+    console.log('+++++ END LOGGING ALLOC DETAILS +++++');
+
```
Comment (ChaiWithJai) [inline ui/tests/acceptance/allocation-detail-test.js:748]: `nit`: Do you want to commit these logs?

**Review by ChaiWithJai — APPROVED** [review]
LGTM

**Pull request merged**